core/scss/components/badge/module.scss

```diff
@@ -1,11 +1,18 @@
-@import '../../tools/functions/core';
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import 'settings';
-
 @import '../../tools/mixins/core';
-@import 'mixins';
 
+// Settings
+@import '../../settings/settings';
+
+// Tools
+@import '../../typography/mixins';
+
+// Generic
+@import '../../generic/global';
+
+// Typography
+@include brand-font-family-ciscoFont;
```

Comment:
Don't use this mixin. It adds *all* of the @font-face rules for all of the Cisco fonts. Instead use the mixin for the individual font(s) that you need.

Reply:
got it